sound/soc/sof/loader.c

```diff
@@ -341,7 +341,7 @@ int snd_sof_run_firmware(struct snd_sof_dev *sdev)
 	if (sdev->first_boot) {
 		ret = snd_sof_debugfs_buf_item(sdev, &sdev->fw_version,
 					       sizeof(sdev->fw_version),
-					       "fw_version");
+					       "fw_version", 0444);
```

Comment:
do you mind adding a side /* read-only */ comment for folks like me who *never* remember what those permissions mean?